server/lib/report_server/reports/report_query.ex

```diff
@@ -51,4 +51,25 @@ defmodule ReportServer.Reports.ReportQuery do
     %{report_query | order_by: new_order_by}
   end
 
+  def get_log_cols(opts) do
+    hide_names = Keyword.get(opts, :hide_names, false)
+    remove_username = Keyword.get(opts, :remove_username, false)
+
+    ["id", "session", "username", "application", "activity", "event", "event_value", "time", "parameters", "extras", "run_remote_endpoint", "timestamp"]
+      |> Enum.filter(&(!(&1 == "username" && remove_username)))
+      |> Enum.map(&(maybe_hash_username(&1 == "username" && hide_names, &1)))
+  end
+
+  def get_log_db_name() do
+    Application.get_env(:report_server, :athena) |> Keyword.get(:log_db_name, "log_ingester_production")
+  end
+
+  defp maybe_hash_username(hash, col) do
+    if hash do
+      hide_username_hash_salt = Application.get_env(:report_server, :athena) |> Keyword.get(:hide_username_hash_salt, "no-hide-username-salt-provided!!!");
+      {"TO_HEX(SHA1(CAST(('#{hide_username_hash_salt}' || \"log\".#{col}) AS VARBINARY)))", col}
+    else
+      {"\"log\".\"#{col}\"", col}
+    end
+  end
```

Comment:
I think the athena data that gets appended has multiple IDs for the user already -- `user_id` and `student_id`.  I suggested in [this document](https://docs.google.com/document/d/16W3Mdq8x7gXs-XilEzic0wIwC4je8tuAyTqiqJ5mrFs/edit?tab=t.0) that it would be nice to standardize on one. Is there a reason that we want username in particular (hashed or not)?

Reply:
I think (at least for now to ease QA) we should have the reports generate the same output so they can be tested against the current reports from the portal to ensure they return the same info.  I think there is going to probably be a pass over the reports near the end of the work to clean them up.